sftp/1.3/modules/ROOT/pages/sftp-read.adoc

```diff
@@ -1,10 +1,29 @@
 = Read a File Using the SFTP Connector - Mule 4
 :page-aliases: connectors::sftp/sftp-read.adoc
 
-The SFTP connector can read a file at any point in the flow, unlike in Mule 3, where the transport can only read files as a result of inbound endpoint polling.
+Anypoint Connector for SFTP (SFTP Connector) can read a file at any point in the flow, unlike in Mule 3, where the transport reads only files as a result of inbound endpoint polling.
 
-Here is the syntax for reading a file:
+The Read operation reads the file in the given path, and returns a Mule message (`MuleMessage`) with the following attributes:
 
+* The file's content as the payload
+* The file's metadata in the message attributes
+* If the file does not exist, you receive an `SFTP:ILLEGAL_PATH` error. Note that the operation does not read directories.
+
+To add the Read operation in Studio, follow these steps:
+
+. In the *Mule Palette* view, search *SFTP* and select the *Read* operation.
+. Drag the *Read* operation onto the Studio canvas.
+. In the *General* tab of the operation configuration screen, click the plus sign (*+*) next to the *Connector configuration* field to access the global element configuration fields.
```

Comment:
I don't remember whether we standardized on "configuration screen" or "properties window." I want to start writing down our decisions because I keep forgetting them.